.travis.yml

```diff
@@ -297,33 +297,20 @@ script:
   - SKIP_FILES3="|shippable.yml|shippable.sh|wercker.yml|wercker.sh|intellij-idea-inspections.xml"
   - SKIP_FILES4="|Jenkinsfile"
   - SKIP_FILES=$SKIP_FILES1$SKIP_FILES2$SKIP_FILES3$SKIP_FILES4
-  - SKIP_CI_BY_COMMIT="false"
+  - source ./.ci/all.sh
   - |
-    if [[ $SKIP_CI_BY_FILES != 'false' ]]; then
-         if [[ $(git diff --name-only HEAD `git merge-base origin/master HEAD` \
-                    | grep -vE "$SKIP_FILES" | cat | wc -c | sed 's/^ *//' ) > 0 ]]; then
-              SKIP_CI_BY_FILES="false"
-         else
-              SKIP_CI_BY_FILES="true"
-         fi
-    fi
-  - |
-    export SKIP_CI_BY_COMMIT="false"
-    if [ $(git log -1 | grep -E "\[maven-release-plugin\] prepare release" | cat | wc -l) -lt 1 ];
-         then
-           SKIP_CI_BY_COMMIT="false"
-         else
-           SKIP_CI_BY_COMMIT="true"
+    if [ -z ${SKIP_CI_BY_FILES+x} ]; then
```

Comment:
Is that `+x` intended as is?

Reply:
That is correct. See https://stackoverflow.com/a/13864829/1016482

Reply:
[gnu:bash:manual](https://www.gnu.org/software/bash/manual/html_node/Shell-Parameter-Expansion.html) agrees on that too.